app/assets/stylesheets/application.css.scss

```diff
@@ -59,6 +59,33 @@
   float: left;
 }
 
+
+.show-header
+{
+  width: 80%;
+  margin: 3% auto;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.